extension/sql/timescale_prometheus.sql

```diff
@@ -0,0 +1,62 @@
+
+CREATE OR REPLACE FUNCTION _prom_catalog.const_support(internal) RETURNS INTERNAL
```

Comment:
Let's not specify a schema for the objects in any of these. Rather the schema will be installed in a particular schema with `CREATE EXTENSION ... WITH SCHEMA '_prom_catalog'`. That way the user of the extension can put it anywhere it wants and there is less coupling between the connector and extension.

Reply:
While I like the idea, I think the has issues with the operator functions: we need to create the operator functions in the extension, because they need superuser, and the connector needs to know their names to be able to define the operators (I don't really want to rely on what happens to postgres when you create multiple of the same operator...)